splunk/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM docker.io/splunk/splunk:latest AS builder
+
+ENV SPLUNK_START_ARGS=--accept-license
+
+EXPOSE 8080
+EXPOSE 8089
+
+USER root
+
+COPY tests/test_logs /opt/splunk/test_logs
+COPY tests/indexes.conf /opt/splunk/etc/apps/search/local/indexes.conf
+COPY scripts/index_verification.sh /opt/splunk/index_verification.sh
+COPY scripts/pre-run.sh /opt/splunk/pre-run.sh
+
+RUN microdnf install -y jq && \
+    microdnf clean all
+
+# USER ansible    # root user is still needed for removing files
```

Comment:
you have to make sure you never try to run splunk as root, because it wasn't meant to run like that

Reply:
as long as we don't need to remove the files, no issues accomplishing that. will fix